.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+
+jobs:
+  docker:
+    name: Build Action Server Docker Image
+    runs-on: ubuntu-latest
+
+    env:
+      DOCKERHUB_USERNAME: oakela
+
+    steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub Registry 🔢
+        run: echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ env.DOCKERHUB_USERNAME }} --password-stdin || true
```

Comment:
why is that piped to true ? Surely if it fails it should fail

Reply:
great question 🤔 copied from rasa repo but i'll remove